client/src/GlobalStyles.js

```diff
@@ -68,6 +68,11 @@ const GlobalStyles = () => {
           background-size: cover;
           background-color: transparent;
         }
+
+        a {
+          text-decoration: none;
+          color: inherit;
+        }
```

Comment:
Is this really part of your PR?

Reply:
Yes. because it is the styling of the text of my button ;)